torchgeo/models/resnet.py

```diff
@@ -7,16 +7,63 @@
 
 import kornia.augmentation as K
 import timm
-import torch.nn as nn
+import torch
 from timm.models import ResNet
 from torchvision.models._api import Weights, WeightsEnum
 
 from ..transforms import AugmentationSequential
 
 __all__ = ["ResNet50_Weights", "ResNet18_Weights"]
 
+
+# https://github.com/zhu-xlab/SSL4EO-S12/blob/d2868adfada65e40910bfcedfc49bc3b20df2248/src/benchmark/transfer_classification/linear_BE_moco.py#L167 # noqa: E501
+# https://github.com/zhu-xlab/SSL4EO-S12/blob/d2868adfada65e40910bfcedfc49bc3b20df2248/src/benchmark/transfer_classification/datasets/EuroSat/eurosat_dataset.py#L97 # noqa: E501
+# Normalization either by 10K or channel-wise with band statistics
```

Comment:
Before we merge this we should get a sense for either which the authors recommend (what do they prefer in their own code or paper or experiments?) or which seems to work better (maybe @calebrob6 can run some experiments with both)

Reply:
authors confirmed that they / 10000 -- https://github.com/zhu-xlab/SSL4EO-S12/issues/2